documentation/intellij.asciidoc

```diff
@@ -22,3 +22,14 @@ You may also supply additional arguments as `devon intellij «args»`. These are
 |`ws-reverse-add`|reverse merge adding new properties
 |`create-script` |create launch script for this IDE, your current workspace and your OS
 |=======================
+
+There are variables that can be used for IntelliJ. These are explained by the following table:
+
+.Variables of devonfw-ide for intelliJ
+[options="header"]
+|=======================
+|*Variable*|*Default-Value*|*Meaning*
+|*`INTELLIJ_VERSION`*|`2019.2`|The version of the tool IntelliJ to install and use.
```

Comment:
adding the default value for the version is dangerous as we will typically not be able to maintain that and forget to update this documentation. So I would rather remove this column or write something like «latest_tested_version» here.